### PR TITLE
Add a test for requestSubmit(null)

### PR DESCRIPTION
**Pull request by rwlbuis**
This should behave exactly as requestSubmit() behaves.

### CODE REVIEW

File: html/semantics/forms/form-submission-0/form-submission-algorithm.html [hunk 1]
```diff
@@ -109,6 +109,19 @@
...
+  let iframe = form.previousSibling;
+  let event;
+  form.addEventListener('submit', e => { event = e; });
+  form.requestSubmit(null);
+  await loadPromise(iframe);
+  assert_true(event.bubbles);
+  assert_true(event.cancelable);
+  assert_equals(event.submitter, null);
+  assert_true(event instanceof SubmitEvent);
+}, 'firing an event named submit; form.requestSubmit(null)');
```
Comment (annevk) [inline html/semantics/forms/form-submission-0/form-submission-algorithm.html:123]: This needs a different name from the test above, no? It seems this would result in a harness error.

Comment (rwlbuis) [reply html/semantics/forms/form-submission-0/form-submission-algorithm.html:123]: It has a different name :) I added null as parameter in the description.

Comment (annevk) [reply html/semantics/forms/form-submission-0/form-submission-algorithm.html:123]: If I search for this exact string, line 110 and 123 light up.

Comment (domenic) [reply html/semantics/forms/form-submission-0/form-submission-algorithm.html:123]: I think the tests are actually identical? Maybe this was already exported somehow?

Comment (annevk) [reply html/semantics/forms/form-submission-0/form-submission-algorithm.html:123]: The original was added in 61c2804bf27a7abc1b8bc406627f6bf3b3344f14...

Comment (rwlbuis) [reply html/semantics/forms/form-submission-0/form-submission-algorithm.html:123]: Ah, I completely missed that somehow! It is good that this is tested, in https://bugs.webkit.org/show_bug.cgi?id=197958 we thought it was not tested. Sorry for the noise!
